tools/sst_dump_tool.cc

```diff
@@ -205,7 +206,38 @@ uint64_t SstFileDumper::CalculateCompressedTableSize(
 int SstFileDumper::ShowAllCompressionSizes(
     size_t block_size,
     const std::vector<std::pair<CompressionType, const char*>>&
-        compression_types) {
+      compression_types,
+    bool show_compression_levels,
+    int32_t compress_level_from,
+    int32_t compress_level_to) {
+
+  fprintf(stdout, "Block Size: %" ROCKSDB_PRIszt "\n", block_size);
+  for (auto& i : compression_types) {
```

Comment:
I think this logic can be simplified by defaulting ```compress_level_from``` and ```compress_level_to``` to ```kDefaultCompressionLevel``` and not requiring ```show_compression_levels``` to be passed.